pkg/compose/create.go

```diff
@@ -1325,6 +1328,7 @@ func (s *composeService) resolveOrCreateNetwork(ctx context.Context, project *ty
 		Attachable: n.Attachable,
 		IPAM:       ipam,
 		EnableIPv6: n.EnableIPv6,
+		EnableIPv4: n.EnableIPv4,
```

Comment:
Oh, missed there were more, I should probably mention this one as well; let me see if I can find a PR with a description

Reply:
cc @robmry if the description looks good 🙈